modules/@apostrophecms/schema/ui/apos/mixins/AposInputMixin.js

```diff
@@ -55,6 +55,11 @@ export default {
     serverError: {
       type: Object,
       required: false
+    },
+
+    noBlurEmit: {
+      type: Boolean,
+      default: false
```

Comment:
New change to allow to cancel blur emit because it brings in the multisite dashboard case.